exchanges/coinbene/coinbene.go

```diff
@@ -0,0 +1,319 @@
+	c.Name = "Coinbene"
+	c.Enabled = false
+	c.Verbose = false
+	c.RESTPollingDelay = 10
+	c.RequestCurrencyPairFormat.Delimiter = ""
+	c.RequestCurrencyPairFormat.Uppercase = true
+	c.ConfigCurrencyPairFormat.Delimiter = "/"
+	c.ConfigCurrencyPairFormat.Uppercase = true
+	c.AssetTypes = []string{ticker.Spot}
+	c.SupportsAutoPairUpdating = true
+	c.SupportsRESTTickerBatching = false
+	c.Requester = request.New(c.Name,
+		request.NewRateLimit(time.Second, 0),
+		request.NewRateLimit(time.Second, 0),
+		common.NewHTTPClientWithTimeout(exchange.DefaultHTTPTimeout))
+	c.APIUrlDefault = coinbeneAPIURL
+	c.APIUrl = c.APIUrlDefault
+	c.Websocket = wshandler.New()
+}
+
+// Setup takes in the supplied exchange configuration details and sets params
+func (c *Coinbene) Setup(exch *config.ExchangeConfig) {
+	if !exch.Enabled {
+		c.SetEnabled(false)
+	} else {
+		c.Enabled = true
+		c.AuthenticatedAPISupport = exch.AuthenticatedAPISupport
+		c.AuthenticatedWebsocketAPISupport = exch.AuthenticatedWebsocketAPISupport
+		c.SetAPIKeys(exch.APIKey, exch.APISecret, "", false)
+		c.SetHTTPClientTimeout(exch.HTTPTimeout)
+		c.SetHTTPClientUserAgent(exch.HTTPUserAgent)
+		c.RESTPollingDelay = exch.RESTPollingDelay
+		c.Verbose = exch.Verbose
+		c.Websocket.SetWsStatusAndConnection(exch.Websocket)
+		c.BaseCurrencies = exch.BaseCurrencies
+		c.AvailablePairs = exch.AvailablePairs
+		c.EnabledPairs = exch.EnabledPairs
+		err := c.SetCurrencyPairFormat()
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = c.SetAssetTypes()
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = c.SetAutoPairDefaults()
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = c.SetAPIURL(exch)
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = c.SetClientProxyAddress(exch.ProxyAddress)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// If the exchange supports websocket, update the below block
+		// err = c.WebsocketSetup(c.WsConnect,
+		//	exch.Name,
+		//	exch.Websocket,
+		//	coinbeneWebsocket,
+		//	exch.WebsocketURL)
+		// if err != nil {
+		// 	log.Fatal(err)
+		// }
+		// c.WebsocketConn = &wshandler.WebsocketConnection{
+		// 		ExchangeName:         c.Name,
+		// 		URL:                  c.Websocket.GetWebsocketURL(),
+		// 		ProxyURL:             c.Websocket.GetProxyAddress(),
+		// 		Verbose:              c.Verbose,
+		// 		ResponseCheckTimeout: exch.WebsocketResponseCheckTimeout,
+		// 		ResponseMaxLimit:     exch.WebsocketResponseMaxLimit,
+		// }
+	}
+}
+
+// FetchTicker gets and stores ticker data for a currency pair
+func (c *Coinbene) FetchTicker(symbol string) (TickerResponse, error) {
+	var t TickerResponse
+	params := url.Values{}
+	params.Set("symbol", symbol)
+	path := fmt.Sprintf("%s%s%s?%s", c.APIUrl, coinbeneAPIVersion, coinbeneFetchTicker, params.Encode())
+	return t, c.SendHTTPRequest(path, &t)
+}
+
+// FetchOrderbooks gets and stores orderbook data for given pair
+func (c *Coinbene) FetchOrderbooks(symbol, size string) (OrderbookResponse, error) {
```

Comment:
For consistency's sake, all other usage of numbers in API funcs (in this case `size`) you use numbers and convert them to strings. Please do that here too